components/raftstore/src/store/worker/read.rs

```diff
@@ -811,6 +879,13 @@ where
                     }
                     _ => unreachable!(),
                 };
+
+                TLS_LOCAL_READ_METRICS.with(|m| m.borrow_mut().local_executed_requests.inc());
```

Comment:
I think these metrics update can be moved to maybe_update_snapshot.
Then no need to handle snap_updated or not

Reply:
What if lease does not pass?

Reply:
> What if lease does not pass?

OK.